sdk/devcenter/azure-developer-devcenter/src/test/java/com/azure/developer/devcenter/generated/EnvironmentsUpdateEnvironmentTests.java

```diff
@@ -0,0 +1,25 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+// Code generated by Microsoft (R) AutoRest Code Generator.
+
+package com.azure.developer.devcenter.generated;
+
+import com.azure.core.http.rest.RequestOptions;
+import com.azure.core.http.rest.Response;
+import com.azure.core.util.BinaryData;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Disabled;
+import org.junit.jupiter.api.Test;
+
+public final class EnvironmentsUpdateEnvironmentTests extends DevCenterClientTestBase {
+    @Test
+    @Disabled
```

Comment:
Did this autogenerate with the `Disabled` annotation?

Reply:
Yes, they are generated (a bit similar to samples).

It serves as a guide on the working tests, providing TestBase with mocked client / recording etc.

But apparently it won't pass as live test directly.